lib/external/math.dart

```diff
@@ -7,6 +7,11 @@ import 'package:Birb/utils/exceptions.dart';
 
 void registerMath(Runtime runtime) {
   registerGlobalFunction(runtime, 'rand', funcRand);
+  registerGlobalFunction(runtime, 'atan2', funcATan2);
+  for(final name in {'sin', 'cos', 'tan'}){
```

Comment:
This maybe saving boilerplate within the `registerMath` function, buts added a lot within funcTrigo. We also style for readable code.

Suggestion, register the functions just as the rest are being registered

Reply:
Alright!